src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -729,10 +729,10 @@ public function get_test_php_version() {
 		$result = array(
 			'label'       => sprintf(
 				/* translators: %s: The recommended PHP version. */
-				__( 'Your site is running a recommended version of PHP (%s)' ),
+				__( 'Your site is running PHP Version (%s).' ),
```

Comment:
I don't think this is simpler:

```suggestion
				__( 'Your site is running PHP %s' ),
```